core/chains/hedera.go

```diff
@@ -0,0 +1,88 @@
+package chains
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"net/http"
+	"sort"
+
+	utils "github.com/xenowits/nakamoto-coefficient-calculator/core/utils"
+)
+
+type Node []struct {
+	Description     string `json:"description"`
+	Node_Account    string `json:"node_account_id"`
+	Stake 			int64  `json:"stake"`
+}
+
+type Link struct{
+	Next	string `json:"next"`
+}
+
+type HederaResponse struct {
+	Nodes	Node
+	Links	Link
+}
+
+func Hedera() (int, error){
+
+	// set base url for requests
```

Comment:
For all godoc, let's use the convention that a comment should start with a capital letter and end with a fullstop.
```suggestion
	// Set base url for requests.
```
The same applies for all comments in this file.